### PR TITLE
fix(accordion): prevent panel icon from shrinking

### LINKED ISSUE #3707: [Accordion] Icon shrinks when texts overflows to multiple lines
example: https://stackblitz.com/edit/react-d3zk7x?file=src/example.js

Screen Shot 2020-08-28 at 2 20 34 PM

### PR DESCRIPTION
**Pull request by maulik1729**
Fixes #3707

#### Description

Give flex none to panel icon to avoid shrink in case of text-overflow

#### Scope
Patch: Bug Fix

### CODE REVIEW

**Comment by chasestarr** [comment]
Hi @maulik1729 can you please share an example displaying why this change is required?

**Comment by maulik1729** [comment]
https://stackblitz.com/edit/react-d3zk7x?file=src/example.js
Icon is flex child it would shrink by default if its sibling overflow.  Icon should not shrink.
@chasestarr

File: src/accordion/styled-components.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ export const ToggleIcon = styled<SharedStylePropsArgT>('svg', props => {
   const {$theme, $disabled, $color} = props;
   return {
     ...getSvgStyles(props),
+    flex: 'none',
```
Comment (chasestarr) [inline src/accordion/styled-components.js:67]: Can you please update to longhand `flex-shrink`? Doing so will help avoid styletron warnings regarding mixed shorthand and longhand properties.

```suggestion
    flexShrink: 0,
```

**Comment by maulik1729** [comment]
@chasestarr Updated PR Please review. Can you please mark this as hacktoberfest-accepted?

**Comment by chasestarr** [comment]
Can you please update jest snapshots?

**Comment by maulik1729** [comment]
Updated test @chasestarr

**Review by chasestarr — APPROVED** [review]
Thank You

**Pull request merged**